di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/utils/JwtBuilder.java

```diff
@@ -67,6 +67,7 @@ public static JWTClaimsSet buildAuthorizationRequestClaims(String errorType) {
                 .claim("response_type", ResponseType.Value.CODE.toString())
                 .claim("redirect_uri", redirectUri)
                 .claim("state", UUID.randomUUID().toString())
+                .claim("persistent_session_id", UUID.randomUUID().toString())
```

Comment:
It's best someone from the ipv-core-team approves this amend
( fyi  @dannyhayes-gds  )

Reply:
@blakeyp Is this related to your PR https://github.com/alphagov/di-ipv-core-back/pull/397 ?

Reply:
So we've added `govuk_signin_journey_id` as a claim here (#138). Looks like this just needs a rebase to fix the conflict now that's been merged

Reply:
Auto merge failed as the pr's had changed the same line, should be good to go now.